docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ConvertType.cs

```diff
@@ -20,17 +20,21 @@ public static void Example()
             var data = mlContext.Data.LoadFromEnumerable(rawData);
 
             // Construct the pipeline.
-            var pipeline = mlContext.Transforms.Conversion.ConvertType("SurvivedInt32", "Survived", DataKind.Int32);
+            var pipeline = mlContext.Transforms.Conversion.ConvertType(
+                "SurvivedInt32", "Survived", DataKind.Int32);
 
             // Let's train our pipeline, and then apply it to the same data.
             var transformer = pipeline.Fit(data);
             var transformedData = transformer.Transform(data);
 
-            // Display original column 'Survived' (boolean) and converted column 'SurvivedInt32' (Int32)
-            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, true);
+            // Display original column 'Survived' (boolean) and converted column 
+            // SurvivedInt32' (Int32)
+            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(
+                transformedData, true);
             foreach (var item in convertedData)
```

Comment:
new line between any two lines that were broken down into multiple lines but no new line between multiple breakdowns of a single line.